Exercise_9/src/MyFramework/customTags/CustomLink.js

```diff
@@ -0,0 +1,19 @@
+const customlink = {
```

Comment:
As I proposed, it is better to have this logic in the "components" directory and rename the variable to `customlinkComponent`.